README.md

```diff
@@ -144,3 +144,115 @@ To configure joypads, use the built-in menu or the `retroarch-joyconfig` command
 ## Compiling and installing
 
 Instructions for compiling and installing RetroArch can be found in the [Libretro/RetroArch Documentation Center](https://docs.libretro.com/).
+
+## CRT SwitchRes
```

Comment:
Is `SwitchRes` some existing name being used? @twinaphex should we instead be using a generic term?